### ISSUE TITLE
Plans to migrate to a supported version of Node?

### ISSUE THREAD

**Issue by michaelmaillot** [pnp/sp-dev-fx-controls-react#1401]
#### Category
[x] Enhancement

[ ] Bug

[x] Question

#### Version

Please specify what version of the library you are using: [3.12.0]

#### Expected / Desired Behavior / Question

As stated in the [documentation](https://pnp.github.io/sp-dev-fx-controls-react/guides/mpa/#install-prerequisites), the current version supported is version 10. But this one's status is currently [EOL](https://github.com/nodejs/release#end-of-life-releases).

Is there any plans to update the repo in order to use at least version 16 of Node (like it's the case for [SPFx v1.15](https://learn.microsoft.com/en-us/sharepoint/dev/spfx/release-1.15#nodejs-v16-support))?

Maybe a v4 is planned?

#### Observed Behavior
N/A

#### Steps to Reproduce
N/A

**Comment by AJIXuMuK** [comment]
@joelfmrodrigues @michaelmaillot - I think we need to change docs to reference to the SPFx versions table.
In that case the valid version on Node could be checked based on SPFx version from the library.

**Comment by michaelmaillot** [comment]
Right, so the table could contain library versions (v1, v2, v3), node versions & SPFx versions?

If that's Ok for everyone, I can update the MPA page accordingly.
